pkg/utils/validator/simple_validator.go

```diff
@@ -37,6 +37,18 @@ func NewResourceStringValidator() *StringValidator {
 	}
 }
 
+// allow small and capital letters, numbers, special characters :.= space
+func NewTlsStringValidator() *StringValidator {
+	re, err := regexp.Compile("^[A-Za-z0-9=:. ]+$")
+	if err != nil {
+		fmt.Printf("Error compiling regex: %v", err)
+		return nil
+	}
+	return &StringValidator{
+		Expression: re,
+	}
+}
+
```

Comment:
Because the --tls-secret flag is the name of an existent kubernetes secret, we won't need this; we need to check that the secret exists in the namespace.

Reply:
removed.     Should we have a validator for the tls-secret name.   I'm not sure what characters are allowed in this name?